account-decoder/src/validator_info.rs

```diff
@@ -1,5 +1,3 @@
-use solana_config_program_client::instructions_bincode::ConfigState;
-
 pub const MAX_SHORT_FIELD_LENGTH: usize = 80;
 pub const MAX_LONG_FIELD_LENGTH: usize = 300;
 pub const MAX_VALIDATOR_INFO: u64 = 576;
```

Comment:
nit: not your fault, but since `MAX_VALIDATOR_INFO` is a bit ambiguous, can you add a comment for this const to say what it's for? ie `/// Max size of a validator info account`